api/app/Enums/Mentorship.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Enums;
+
+use App\Traits\HasLocalization;
+
+enum Mentorship
+{
+    use HasLocalization;
+
+    case MENTOR;
+    case MENTEE;
```

Comment:
Might be worth adding this to our dictionary since mentee is a real word.

Reply:
Good call, added in [add mentee to dictionary](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12334/commits/9f63e141b759a402df2d5326f74a2d469b38e12a)